WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/SelectionList.swift

```diff
@@ -10,18 +10,27 @@ struct SelectionList<T: Hashable>: View {
     /// Key path to find the content to be displayed
     private let contentKeyPath: KeyPath<T, String>
 
+    /// Key path to find the value to be binded for selection
+    private let selectionKeyPath: KeyPath<T, U>
+
```

Comment:
nit: You mentioned in the PR description that you decided not to use this but left it in case it's needed later. I'd suggest going ahead and removing it for now, since it isn't being used, and we can always add it back later if we decide it's needed. (Not a big deal but it keeps things cleaner.)

Reply:
This makes sense, I've removed it in 8651afc5ffece50db9fa24632439daf234a15dfb